spec/controllers/controller_remember_me_spec.rb

```diff
@@ -58,6 +72,22 @@
       expect(cookies.signed['remember_me_token']).to eq assigns[:user].remember_me_token
     end
 
+    it 'login(email,password,remember_me) logs user in and remembers with custom cookie name' do
+      sorcery_controller_property_set(:remember_me_cookie_name, :custom)
+
+      expect(User).to receive(:authenticate).with('bla@bla.com', 'secret', '1') { |&block| block.call(user, nil) }
+      expect(user).to receive(:remember_me!)
+      expect(user).to receive(:remember_me_token).and_return('abracadabra').twice
```

Comment:
Why does this receive the token twice? I'm not very familiar with the callback system, but this looks like it might be a side-effect of the double callback bug in `0.13.0`

Reply:
@athix to be honest, I copied it over from the testcase above this one. So it could be part of the double callback bug. When I remove the `twice` method though, it gives an error. Could it be that the double callback bug is still relevant?